salt/utils/network.py

```diff
@@ -973,14 +973,17 @@ def ip_addrs6(interface=None, include_loopback=False, interface_data=None):
 def hex2ip(hex_ip, invert=False):
     '''
     Convert a hex string to an ip, if a failure occurs the original hex is
-    returned
+    returned. If 'invert=True' assume that ip from /proc/net/<proto>
     '''
     if len(hex_ip) == 32:  # ipv6
         ip = []
         for i in range(0, 32, 8):
             ip_part = hex_ip[i:i + 8]
             ip_part = [ip_part[x:x + 2] for x in range(0, 8, 2)]
-            ip.append("{0[3]}{0[2]}:{0[1]}{0[0]}".format(ip_part))
+            if invert:
+                ip.append("{0[3]}{0[2]}:{0[1]}{0[0]}".format(ip_part))
```

Comment:
Given that the default for `invert` is `False`, doesn't this imply a change in behavior here? Is this a bug fix or was this unintentional?